code/modules/tgui/modules/power_monitor.dm

```diff
@@ -0,0 +1,54 @@
+/datum/tgui_module/power_monitor
+	name = "Power monitor"
+	var/select_monitor = FALSE
```

Comment:
Could you document this var? I think It means that you'll view the powernet selection screen on the PDA but I had to dig around a bit.

Reply:
I'll do a mass documentation when this is in the ready phase